node/src/main/scala/com/wavesplatform/network/HandshakeHandler.scala

```diff
@@ -31,12 +31,12 @@ class HandshakeDecoder(peerDatabase: PeerDatabase) extends ReplayingDecoder[Void
 
 case object HandshakeTimeoutExpired
 
-class HandshakeTimeoutHandler(handshakeTimeout: FiniteDuration) extends ChannelInboundHandlerAdapter with ScorexLogging {
+class HandshakeTimeoutHandler(handshakeTimeout: => FiniteDuration) extends ChannelInboundHandlerAdapter with ScorexLogging {
```

Comment:
Better do `handshakeTimeout` an explicit function 
```suggestion
class HandshakeTimeoutHandler(handshakeTimeout: () => FiniteDuration) extends ChannelInboundHandlerAdapter with ScorexLogging {
```

Reply:
Actually, looks like there's no need to use a by-name parameter at all here.